style.css

```diff
@@ -0,0 +1,321 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  background-color: rgb(238, 238, 238);
+  font-family: 'Maven Pro', sans-serif;
+}
+
+.overline {
+  text-decoration: line-through;
+}
+
+a {
+  color: gray;
+  text-decoration: none;
+}
+
+.flex {
+  display: flex;
+}
+
+.white {
+  color: #fff;
+}
+
+.gray {
+  color: #9d9999;
+}
+
+.orange {
+  color: #f42;
+}
+
+.fsmall {
+  font-size: 12px;
+}
+
+.thin {
+  font-family: 'Heebo', sans-serif;
+  font-size: 14px;
+
+  /* padding: 5px; */
+}
+
+.orange-headers {
+  font-size: 30px;
+  color: rgb(241, 106, 57);
+  font-weight: 700;
+}
+
+.mobile-nav {
+  display: none;
+}
+
+ul {
+  list-style: none;
+}
+
+.nav2 .nav-search {
+  padding-top: 20px;
+  padding-right: 15px;
+  text-align: center;
+}
+
+.nav-search:hover {
+  border: solid 1px darkgray;
+}
+
+.news-header {
+  margin: 20px;
+}
+
+section {
+  display: grid;
+  grid-template-columns: 70% 30%;
+  grid-template-rows: 1fr 1fr;
+  gap: 10px;
+  grid-template-areas:
+    "section1 section2"
+    "section1 section3";
+}
+
+.section1 {
+  grid-area: section1;
+  max-height: 380px;
+  background:
+    linear-gradient(
+      180deg,
+      rgba(148, 67, 20, 0.6) 30%,
+      rgba(0, 0, 0, 1) 100%
+    ),
+    url('img/bitcoin.jpg');
+  background-size: cover;
+  background-repeat: no-repeat;
+  flex-direction: column;
+  justify-content: space-between;
+}
+
+.section1 img {
+  max-width: 75vh;
+  align-self: flex-start;
+  padding: 25px;
+}
+
+.showcase-title {
+  font-size: 40px;
+  font-weight: 700;
+}
+
+.section1 ul,
+.section2 ul,
+.section3 ul {
+  list-style: none;
+}
```

Comment:
Same style definition of the selector on line 57. Group selectors, or create a new class.